android/src/main/java/vn/hunghd/flutterdownloader/DownloadWorker.java

```diff
@@ -126,6 +126,10 @@ private void startBackgroundIsolate(Context context) {
                 // lookup will fail.
                 FlutterCallbackInformation flutterCallback =
                         FlutterCallbackInformation.lookupCallbackInformation(callbackHandle);
+                if (flutterCallback == null) {
```

Comment:
Hi, just a quick question if you got the time. 
If the flutterCallback turns out to be null, what are we expecting to happen on devices? Are we expecting the downloader to not do anything?

Reply:
Yes